src/univariate/continuous/weibull.jl

```diff
@@ -139,3 +139,41 @@ end
 #### Sampling
 
 rand(rng::AbstractRNG, d::Weibull) = xv(d, randexp(rng))
+
+#### Fit model
+
+"""
+    fit_mle(::Type{<:Weibull}, x::AbstractArray{T}; 
+    α0::Float64 = 1.0, maxiter::Int = 1000, tol::Float64 = 1e-16)
+
+Maximum Likelihood Estimate of `Weibull` Distribution via Newton's Method
+
+"""
+function fit_mle(::Type{<:Weibull}, x::AbstractArray{T};
+    α0::Float64 = 1.0, maxiter::Int = 1000, tol::Float64 = 1e-16) where T <: Real
+
+    ϵ = Inf
+    N = 0
+
+    α = α0
+
+    lnx = log.(x)
```

Comment:
No need for broadcasting here:

```suggestion
    lnx = map(log, x)
```